Assets/Scripts/Gameplay/GameplayObjects/ServerTossAttack.cs

```diff
@@ -0,0 +1,103 @@
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    public class ServerTossAttack : NetworkBehaviour
```

Comment:
We have the TossAction and the TossAttack.
From what I'm reading here, this is the bomb right? Why not just call it bomb?

Reply:
Seemed like a friendlier name 😅.

Reply:
same comment as above, _attack_ sounds like this is a command or an action, when this code describes the behaviour of an object moving around, not an action. The action is to spawn that object, but then the object itself has it's own behaviour here.